ydb/core/grpc_services/rpc_export.cpp

```diff
@@ -511,10 +524,27 @@ class TExportRPC: public TRpcOperationRequestActor<TDerived, TEvRequest, true>,
         }
     }
 
+    bool HasCommonSourcePathPrefix(const TStringBuf path) const {
+        return path.StartsWith(CommonSourcePath) && path.size() > CommonSourcePath.size() && path[CommonSourcePath.size()] == '/';
```

Comment:
Так лучше считывается:
```suggestion
        return path.StartsWith(CommonSourcePath)
            && path.size() > CommonSourcePath.size()
            && path[CommonSourcePath.size()] == '/';
```